apps/trading/components/welcome-dialog/risk-message.tsx

```diff
@@ -0,0 +1,42 @@
+import { t } from '@vegaprotocol/i18n';
+import { ExternalLink, Icon } from '@vegaprotocol/ui-toolkit';
+import { Links, Routes } from '../../pages/client-router';
+
+export const RiskMessage = () => {
+  return (
+    <>
+      <div className="bg-neutral-100 dark:bg-neutral-800 dark:text-neutral-200 p-6 mb-6">
+        <ul className="list-disc ml-6 text-lg">
+          <li>
+            {t(
+              'No party hosts or operates this IFPS website or offers any financial advice.'
```

Comment:
This has been cut from the text. Perhaps you can replace it with the following, depending on where it turns up/what it's for: 
 "This website is hosted on a decentralised network, the Interplanetary File System (“IPFS”).  The IPFS decentralised web is made up of all the computers (nodes) connected to it.  Data is therefore stored on many different computers."

Reply:
Removing this line as discussed